src/model/Column.ts

```diff
@@ -386,6 +387,13 @@ export default class Column extends AEventDispatcher {
   }
 
   rendererType(): string {
+     if(this.desc.type.constructor === Array) {
```

Comment:
how can the `desc.type` be an array? per definition it is a single string determining the column model type